platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/service/schemas/V0540PlatformStateSchema.java

```diff
@@ -24,11 +26,16 @@
  * Defines the {@link PlatformState} singleton and initializes it at genesis.
  */
 public class V0540PlatformStateSchema extends Schema {
+
     private static final Function<Configuration, SemanticVersion> UNAVAILABLE_VERSION_FN = config -> {
         throw new IllegalStateException("No version information available");
     };
 
     public static final String PLATFORM_STATE_KEY = "PLATFORM_STATE";
+    // State ID must match virtual_map_state.proto
+    public static final int PLATFORM_STATE_STATE_ID = 26;
```

Comment:
Can we add a reference to the actual proto ordinal here? Anyway platform depends on hapi now and this schema is subject for refactoring, which will be done soon.

Reply:
OK